### PR TITLE
Add BITalino BLE Board to Brainflow

### PR DESCRIPTION
**Pull request by alchemy-abc**
BITalino BLE Board with 100Hz sample rate and 2 EEG channels.

### CODE REVIEW

**Comment by Andrey1994** [comment]
Hi!

Could you send a link to exact bitalino device(and maybe firmware\config) which was used for it?

File: src/board_controller/inc/brainflow_boards.h [hunk 1]
```diff
@@ -222,13 +215,18 @@ json brainflow_boards_json = {
             {"num_rows", 10},
             {"eeg_channels", {1, 2, 3, 4, 5, 6, 7, 8}}
         }},
-        {"16",
```
Comment (Andrey1994) [inline src/board_controller/inc/brainflow_boards.h:225]: looks like you didnt resolve merge conflicts properly, in fact you removed one of supported boards... and there are more places like this

I think I will get commits from this PR, push them to another branch in brainflow-dev and fix such issues by myself when I get bitalino device

**Comment by Andrey1994** [comment]
thanks!

It looks ok, but there are merge conflicts which need to be solved. I will close and reopen this PR from another branch and fix by myself

File: src/board_controller/inc/brainflow_boards.h (hunk 1 above)
Comment (alchemy-abc) [reply src/board_controller/inc/brainflow_boards.h:225]: when I add the Bitalino board there was not gforcepro in the repository, sorry for this conflict
we can fix it. Is 18 good for boardId?

Comment (Andrey1994) [reply src/board_controller/inc/brainflow_boards.h:225]: Yeah, 18 is fine. That would be great if you fix it. Create a PR to bitalino branch instead master

Comment (alchemy-abc) [reply src/board_controller/inc/brainflow_boards.h:225]: Ok, I do it, now

Comment (Andrey1994) [reply src/board_controller/inc/brainflow_boards.h:225]: Thanks!

**Pull request closed without merging**